spec/requests/api/v1/add_ons_spec.rb

```diff
@@ -87,22 +85,19 @@
     it 'returns a add-on' do
       get_with_token(
         organization,
-        "/api/v1/add_ons/#{add_on.code}"
+        "/api/v1/add_ons/#{add_on.code}",
       )
 
       expect(response).to have_http_status(:success)
-
-      result = JSON.parse(response.body, symbolize_names: true)[:add_on]
-
-      expect(result[:lago_id]).to eq(add_on.id)
-      expect(result[:code]).to eq(add_on.code)
+      expect(json[:add_on][:lago_id]).to eq(add_on.id)
+      expect(json[:add_on][:code]).to eq(add_on.code)
     end
 
     context 'when add-on does not exist' do
       it 'returns not found' do
         get_with_token(
           organization,
-          "/api/v1/add_ons/555"
+          "/api/v1/add_ons/555",
```

Comment:
Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. (https://rubystyle.guide#consistent-string-literals)